pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -170,6 +172,22 @@ func (pas *PodAutoscalerStatus) IsInactive() bool {
 	return pas.GetCondition(PodAutoscalerConditionActive).IsFalse()
 }
 
+// HasBeenActive returns true if the pod autoscaler has reached its initial scale.
+func (pas *PodAutoscalerStatus) HasBeenActive() bool {
+	if val, ok := pas.Annotations[initiallyActiveAnnotation]; !ok || val != "true" {
+		return false
+	}
+	return true
+}
+
+// MarkInitiallyActive marks the PA's PodAutoscalerConditionInitiallyActive condition true.
+func (pas *PodAutoscalerStatus) MarkInitiallyActive() {
```

Comment:
Should we update this and the annotation to match the getter (i.e. `MarkHasBeenActive` etc)?